mappings/net/minecraft/util/collection/DataPool.mapping

```diff
@@ -10,4 +10,9 @@ CLASS net/minecraft/unmapped/C_owfsqzhr net/minecraft/util/collection/DataPool
 	METHOD m_sepjgpgp wrapCodecToAllowEmpty (Lcom/mojang/serialization/Codec;)Lcom/mojang/serialization/Codec;
 	CLASS C_eaytywro Builder
 		FIELD f_gfrmxqai entries Lcom/google/common/collect/ImmutableList$Builder;
+		METHOD m_djjyqyvx addData (Ljava/lang/Object;)Lnet/minecraft/unmapped/C_owfsqzhr$C_eaytywro;
```

Comment:
```suggestion
		METHOD m_djjyqyvx add (Ljava/lang/Object;)Lnet/minecraft/unmapped/C_owfsqzhr$C_eaytywro;
```
more concise, you know you're building a `DataPool` here so we don't really want to add the same `data` name twice